app/workers/send_eoc_deadline_reminder_email_to_candidates_worker.rb

```diff
@@ -0,0 +1,13 @@
+class SendEocDeadlineReminderEmailToCandidatesWorker
+  include Sidekiq::Worker
+
+  def perform
+    GetApplicationsApproachingFirstEocDeadline.call.each do |application|
+      SendEocFirstDeadlineReminderEmailToCandidate.call(application_form: application)
+    end
+
+    GetApplicationsApproachingSecondEocDeadline.call.each do |application|
+      SendEocSecondDeadlineReminderEmailToCandidate.call(application_form: application)
+    end
+  end
+end
```

Comment:
How do we plan to execute this worker? Does it need a clock entry or is it to be run manually on each of the four days?

Reply:
@duncanjbrown was it decided that we would run this manually?